### ISSUE TITLE
PostgreSQL

### ISSUE THREAD

**Issue by jgartee** [AnimalRescue/AnimalRescue#55]
Parent [#52]
If we consider A+ES as a modeling implementation under a DDD architecture, it appears that Vince Vaughn has studied a number of databases for applicability.  https://vaughnvernon.co/?p=942

Let's discuss the advantages and disadvantages as well as alternatives.

**Comment by jgartee** [comment]
## Advantages
- Has Ubuntu implementation and support
- Can be handled as serialized JSON
- Is an ACID database - https://en.wikipedia.org/?title=ACID
- Community-based and free - http://www.postgresql.org/
- Has reporting tool support - http://www.postgresql.org/download/products/5-reporting-tools/

## Disadvantages
- Haven't used it before
- Haven't found any good reporting tools for the database yet

**Comment by jgartee** [comment]
Kim's preferred method of reporting is to download a csv and mangle it herself.  However, Kim may not be the only user and it is possible we'll need standard reports over time.

**Comment by ghost** [comment]
Does this mean we are moving away from Entity Framework?

**Comment by jgartee** [comment]
Not necessarily.  We'll have to weigh the options.  We haven't even discussed whether or not A+ES is even feasible for our implementation.  I don't want to make any design decisions until we absolutely have to.
